script/build.sh

```diff
@@ -168,3 +169,27 @@ if [ "$1" == "sp1" ]; then
         fi
     fi
 fi
+
+# TDX
+if [ "$1" == "tdx" ]; then
+    check_toolchain $TOOLCHAIN_TDX
+    if [ -n "${CLIPPY}" ]; then
+        cargo ${TOOLCHAIN_TDX} clippy -p raiko-host -p tdx-prover -F "tdx enable" -F $TASKDB -- -D warnings
```

Comment:
Is think `-F $TASKDB` is not longer needed here and in next lines, this feature no longer exists 
```suggestion
        cargo ${TOOLCHAIN_TDX} clippy -p raiko-host -p tdx-prover -F "tdx enable"  -- -D warnings
```

Reply:
Yes! This was left from the `main-old` -> `main` rebase. Fixed in [99c350c](https://github.com/NethermindEth/raiko/pull/27/commits/99c350cf77a8cdde25be0008ecc58093e46a2a68)